Samples/SampleCode/test.py

```diff
@@ -0,0 +1,108 @@
+import os
+import tempfile
+import vtk
+from vtk.util.keys import StringKey
+from vtk.vtkCommonCore import vtkInformationIterator
+import vtku3dexporterPython
+
+
+def write_u3d(file_path, actors):
+    render_window = vtk.vtkRenderWindow()
+    render_window.OffScreenRenderingOn()
+    renderer = vtk.vtkRenderer()
+    render_window.AddRenderer(renderer)
+
+    for actor in actors:
+        renderer.AddActor(actor)
+
+    renderer.ResetCamera()
+
+    u3d_exporter = vtku3dexporterPython.vtkU3DExporter()
+    u3d_exporter.SetFileName(file_path)
+    u3d_exporter.SetInput(render_window)
+    u3d_exporter.Write()
+
+
+def create_actor_from_stl(path):
+    assert os.path.exists(path)
+    reader = vtk.vtkSTLReader()
+    reader.SetFileName(path)
+
+    mapper = vtk.vtkPolyDataMapper()
+    mapper.SetInputConnection(reader.GetOutputPort())
+
+    actor = vtk.vtkActor()
+    actor.SetMapper(mapper)
+    return actor
+
+
+def set_name_for_actor(name, actor):
+    """
+    Sets the name in the PropertyKeys of a vtkActor
+    """
+    key = StringKey.MakeKey("MeshName", "root")
```

Comment:
as I understand it, location should be the name of the code entity that has the key instance. so in this case it should be `test.set_name_for_actor`, correct? even if it doesnt affect functionality at all